packages/transition-backend/src/services/nodes/NodeCollectionUtils.ts

```diff
@@ -79,3 +79,45 @@ export const saveAndUpdateAllNodes = async (
     await Promise.allSettled(addPromises);
     progressEmitter?.emit('progress', { name: 'UpdatingTransferableNodes', progress: 1.0 });
 };
+
+/**
+ * Save the nodes to cache including the transferable node
+ * We do the saving in batch, to not overload json2capnp
+ *
+ * @param nodeCollection The collection containing the nodes to update and save
+ * @param collectionManager The collection manager, used to create the nodes
+ */
+export const saveAllNodesToCache = async (
+    nodeCollection: NodeCollection,
+    placeCollection: PlaceCollection | undefined,
```

Comment:
I don't think the collection is necessary in this case either

Reply:
You mean the placeCollection?

Reply:
yes, in fact, I am sure it is not necessary